Index.html

```diff
@@ -0,0 +1,282 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="Style.css" />
+    <title>Portfolio</title>
+  </head>
+
+  <body>
+    <div class="container">
+      <section class="mobile-container">
+        <header class="topnav">
+          <nav>
+            <a href="#home" class="active">My Logo</a>
+            <a class="icon"><img src="images/btn.png" /></a>
+          </nav>
+        </header>
+        <main class="first-section">
+          <h1 class="headline-1">Hey There. I’m Yaser.</h1>
+          <h1 class="title-1">I'm a Software Developer</h1>
+          <p class="text-content">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p>
+          <ul class="fa-icon">
+            <li>
+              <a href="#"><img src="images/github.svg" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/Linkedin icon.svg" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/angel list.svg" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/twitter.svg" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/medium.svg" /></a>
+            </li>
+          </ul>
+        </main>
+      </section>
+      <section>
+        <main>
+          <h1 class="headline-2">My Recent Works</h1>
+          <hr class="h-row" />
+          <div class="img-1"></div>
+          <h1 class="headline-3">Multi-Post Stories</h1>
```

Comment:
- [ ] `h1` tag is for the main title `Hey There, I'm Yaser.` And `h1` tag only appears one time per webpage. It would be better to use other `hx` tags and maintain the hierarchy.  Kindly fix this issue.